src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/MessageProcessorServiceImpl.java

```diff
@@ -52,24 +51,24 @@ public void processMessage(String consumerId,
             chipsRestClient.sendToChips(message, consumerId);
         } catch (HttpStatusCodeException hsce) {
             logMap.put("HTTP Status Code", hsce.getStatusCode().toString());
-            handleFailedMessage(message, hsce, logMap, failedMessageIds);
+            return handleFailedMessage(message, hsce, logMap);
```

Comment:
The handleFailedMessages should just 'deal' with them and doesn't need to return the true/false. It would be better to have

handleFailedMessage(message, hsce, logMap);
return false;

in each catch.

Reply:
so basically as it was for the catch blocks but just add a return false; into each catch block

Reply:
We only want to return false if all retires are exhuasted.

Reply:
sorry, ignore above, i was thinking in terms of an exceptin being a failure but it is when the message is placed on retry which happens in handleFailedMessages.